core/designsystem/src/main/java/com/iteneum/designsystem/components/phonenumbertext/PhoneNumberFormat.kt

```diff
@@ -0,0 +1,44 @@
+package com.iteneum.designsystem.components.phonenumbertext
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.input.OffsetMapping
+import androidx.compose.ui.text.input.TransformedText
+
+/*This block of code give format
+* for number phone in USA format
+* */
+const val mask = "xxx xxx xxxx"
+fun mobileNumberFilter(text: AnnotatedString): TransformedText {
```

Comment:
Can you tranform this function to an extension function, please?

Reply:
We are going to create another task to create this extension function.
I talked with the scrum master and the architect.